docs/image-config/README.md

```diff
@@ -0,0 +1,142 @@
+# Configuring the K8ssandra images
+
+Often users find it neccesary to configure various properties of the images they are using, this may be to:
+
+* Use an internal image registry to avoid dependancy on Docker Hub.
+* Use a custom image (for example to customise aspects of the classpath or run customised versions of Cassandra).
+
+## cass-operator images
+
+The cassandra pods have containers for Cassandra (including the management API), Medusa, config-builder and system-server-logger. These pods are under the management of cass-operator.
```

Comment:
Medusa is not under management of cass-operator.

Reply:
Oop, that's true. I get confused because the container is in Cassandra's pod. I'll fix, thanks.